Missionframework/modules/26_cratefiller/fnc/fn_cratefiller_getNearStorages.sqf

```diff
@@ -30,44 +30,40 @@ lbClear _ctrlStorage;
 private _nearFOB = [] call KPLIB_fnc_common_getPlayerFob;
 private _type = objNull;
 private _config = "";
-private _number= 0;
+private _number = 0;
 private _index = 0;
 private _picture = "";
-private _storages = [];
 private _blacklist = [
     KPLIB_preset_crateSupplyE,
     KPLIB_preset_crateSupplyF,
     KPLIB_preset_crateAmmoE,
     KPLIB_preset_crateAmmoF,
     KPLIB_preset_crateFuelE,
     KPLIB_preset_crateFuelF,
-    KPLIB_logistic_building
+    KPLIB_logistic_building,
+    "GroundWeaponHolder",
+    "WeaponHolderSimulated",
+    "#slop",
+    "#track",
```

Comment:
maybe instead of doing that you can ignore all classes starting with `#` ? There are other engine objects that have similar names. eg. `#soundsource` and it's possible there is more of them.

Reply:
Is it possible to do a check for "#..."?

Reply:
`"str" select 0 != "#" `

Reply:
`forEach (((getMarkerPos _nearFOB) nearObjects KPLIB_param_fobRange) select {!(typeOf _x in _blacklist) || !((_x select 0) != "#")});`
This should do the job?!

Reply:
I think so. Maybe put all objects in variable first, all of it inline is not very readable.

Reply:
I'll do a short performance check 👍